Makefile

```diff
@@ -32,7 +33,7 @@ output/docs.pdf: $(DOC_FILES)
 	-v $(shell pwd)/:/input/:ro \
 	-v $(shell pwd)/output/:/output/ \
 	-u $(shell id -u) \
-	vbatts/pandoc -f markdown_github -t latex -o /$@ $(patsubst %,/input/%,$(DOC_FILES))
+	$(PANDOC) -f markdown_github -t latex -o /$@ $(patsubst %,/input/%,$(DOC_FILES))
```

Comment:
I'm just curious before any test or investigation, $(PANDOC) is a **path to the pandoc executive** when default, but we expect **pandoc image** here, how this replacement works?